parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+echo "Sourcing cse_install_distro.sh for Mariner"
+
+removeContainerd() {
+    retrycmd_if_failure 10 5 60 dnf remove -y moby-containerd
+}
+
+installDeps() {
+    dnf_makecache || exit $ERR_APT_UPDATE_TIMEOUT
+    dnf_update || exit $ERR_APT_DIST_UPGRADE_TIMEOUT
+    for apt_package in ca-certificates cifs-utils cracklib ebtables ethtool fuse git iotop iproute ipset iptables jq pam nfs-utils socat sysstat traceroute util-linux xz zip; do
+      if ! dnf_install 30 1 600 $apt_package; then
+        exit $ERR_APT_INSTALL_TIMEOUT
+      fi
+    done
+}
+
+installGPUDrivers() {
+    echo "GPU drivers not yet supported for Mariner"
+    exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
+}
+
+installSGXDrivers() {
+    echo "SGX drivers not yet supported for Mariner"
+    exit $ERR_SGX_DRIVERS_START_FAIL
+}
+
+# CSE+VHD can dictate the containerd version, users don't care as long as it works
+installStandaloneContainerd() {
+    # azure-built runtimes have a "+azure" suffix in their version strings (i.e 1.4.1+azure). remove that here.
+    CURRENT_VERSION=$(containerd -version | cut -d " " -f 3 | sed 's|v||' | cut -d "+" -f 1)
+    # v1.4.1 is our lowest supported version of containerd
+    local CONTAINERD_VERSION="1.4.3"
```

Comment:
is this the rpm repo we'll be using? https://packages.microsoft.com/cbl-mariner/1.0/prod/base/x86_64/rpms/

I only see moby-containerd-1.3.7.

Reply:
Yes that's the repo. I've got a task to bring in a newer version of containerd in the next iteration. For the time being the dnf install below this isn't enforcing the specific version request but I'll re-add it once the newer version is available.

Reply:
Create a work item to track the work so we don't forget https://msazure.visualstudio.com/CloudNativeCompute/_workitems/edit/9443959